tornado_elasticsearch.py

```diff
@@ -201,6 +201,9 @@ class for serialization
                                                               ignore=ignore)
                 (status, headers, data) = result
             except TransportError as e:
+                if method == 'HEAD' and e.status_code == 404:
+                    raise gen.Return(False)
```

Comment:
Why is `perform_request` returning a boolean? It's job is to raise errors when things go wrong while making a request.